src/Context/UserContext.php

```diff
@@ -13,6 +13,34 @@ class UserContext extends RawWordpressContext
 {
     use Traits\UserAwareContextTrait, Traits\CacheAwareContextTrait;
 
+    /**
+     * Verify a username is valid and get the matching user information.
+     *
+     * @param string $username
+     *
+     * @throws \RuntimeException
+     *
+     * @return array The WordPress user details matching the given username.
+     */
+    protected function getUserByName(string $username)
```

Comment:
I'm not sure this method is needed - an existing method might be able to be used here.